rs/nns/integration_tests/test_canisters/governance_mem_test_canister.rs

```diff
@@ -314,6 +314,9 @@ fn allocate_neuron(id: u64) -> Neuron {
         neuron_type: None,
         visibility: None,
         voting_power_refreshed_timestamp_seconds: None,
+        // These are ignored, because they are derived.
```

Comment:
Nit. Could you mention in the comment which operation is deriving them?

Reply:
Not exactly sure what you mean...

Internally, whenever (native) Neuron is converted to NeuronProto (defined in governance.proto), it always populates these fields. Then, whenever NeuronProto is converted to api Neuron (defined in the rs/nns/governance/api dir), these are simply copied. But (native) Neuron is an internal concept -> I don't see how this explanation would relate to what this comment currently says. Are you just saying that the comment should (additionally) say (something to the effect of)

> These are populated when converting from the native internal Neuron to Neuron as defined by governance.proto

?

Currently, you only need a few pieces of information to derive these:

1. other neuron fields:
  a. stake - This is the "base" of voting power. The rest is "just" bonus multipliers.
    i. cached_stake_e8s
    ii. fees_e8s
    iii. staked_maturity_e8s_equivalent
  b. aging since
  c. dissolve delay
2. the current time

(in the future, you will also need some settings related to what happens when voting power has not been "refreshed" in a "long" time, but those can still be regarded as "constants", since they would almost certainly still be pretty static.)

Reply:
We discussed over zoom, the code populating those fields is `Neuron::into_proto`

Reply:
Discussed in Zoom. Yes, what Arshavir was asking for is to add something like "These get derived in Neuron::into_proto.". In Zoom, I said, I would include this change as part of this PR. Arshavir said this is not important enough to invalidate passing tests. Therefore, I now change my mind: I will make this change, but in another PR. That way, this PR can be merged right away.